sec1_intro_and_setup/test_app/config/application.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "boot"
```

Comment:
Whats this file for?

Reply:
This file allows you to pulls all gems you required in the gem file into your project.

Reply:
By using `Bundler`. `Bundler` will look in your `Gemfile` for gems belonging to the groups you specified in the `Gemfile`, and call `require` on each of the gems it finds.